src/modules/aws.rs

```diff
@@ -44,42 +44,34 @@ fn get_aws_region_from_config(aws_profile: &Option<String>) -> Option<String> {
     Some(region.to_string())
 }
 
-fn get_aws_region() -> Option<(String, String)> {
-    env::var("AWS_DEFAULT_REGION")
-        .ok()
-        .map(|region| (String::new(), region))
-        .or_else(|| {
-            let aws_profile = env::var("AWS_PROFILE").ok();
-            let aws_region = get_aws_region_from_config(&aws_profile);
-
-            if aws_profile.is_none() && aws_region.is_none() {
-                None
-            } else {
-                Some((
-                    aws_profile.unwrap_or_default(),
-                    aws_region.unwrap_or_default(),
-                ))
-            }
-        })
-        .or_else(|| {
-            env::var("AWS_REGION")
-                .ok()
-                .map(|region| (String::new(), region))
-        })
+fn get_aws_region() -> (String, String) {
```

Comment:
I think it can be a little clearer that this function returns a tuple of `(profile, region)`, either by using named values or with a comment.

Reply:
I used alias typed (which I never used before).
Was it what you meant?

As for commenting the code, I need to take a look at the rest of the project to have an hint about the "style" used.